### PR TITLE
Migrate to pytest-selenium and use personatestuser.org

### PR DESCRIPTION
**Pull request by davehunt**
This can be reviewed, however pytest-selenium is not yet released so we may want to hold of merging it until we're ready.

### CODE REVIEW

**Comment by davehunt** [comment]
Adhoc run: http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/bidpom.adhoc.pytest-selenium/7/

We still shouldn't merge this yet, but it's looking good!

File: tests/check_add_email.py [hunk 1]
```diff
@@ -14,44 +15,36 @@
...
+        selenium.find_element(*self._persona_login_button_locator).click()
 
         from browserid.pages.sign_in import SignIn
-        signin = SignIn(mozwebqa.selenium, mozwebqa.timeout)
+        signin = SignIn(selenium, timeout)
         signin.click_add_another_email_address()
-        signin.new_email = user.additional_emails[0]
-        assert signin.new_email == user.additional_emails[0], "new email getter failed"
+        signin.new_email = new_email
+        assert signin.new_email == new_email, 'new email getter failed'
```
Comment (bobsilverberg) [inline tests/check_add_email.py]: Do we really need this?

File: tests/check_sign_in.py [hunk 2]
```diff
@@ -15,117 +15,89 @@
...
-
-    def test_sign_in(self, mozwebqa):
+    def test_sign_in(self, selenium, timeout, verified_user):
         from browserid.pages.sign_in import SignIn
-        signin = SignIn(mozwebqa.selenium, mozwebqa.timeout)
-        signin.email = mozwebqa.email
-        assert signin.email == mozwebqa.email, "email getter failed"
+        signin = SignIn(selenium, timeout)
+        signin.email = verified_user['email']
+        assert signin.email == verified_user['email']
```
Comment (bobsilverberg) [inline tests/check_sign_in.py]: Not sure that we need this.

File: tests/check_sign_in.py [hunk 3]
```diff
@@ -15,117 +15,89 @@
...
         signin.click_next(expect='password')
-        signin.login_password = mozwebqa.password
-        assert signin.login_password == mozwebqa.password, "password getter failed"
+        signin.login_password = verified_user['password']
+        assert signin.login_password == verified_user['password']
```
Comment (bobsilverberg) [inline tests/check_sign_in.py]: Or this.

File: tests/check_sign_in.py [hunk 4]
```diff
@@ -15,117 +15,89 @@
...
-        signin.email = user.primary_email
+        signin = SignIn(selenium, timeout)
+        signin.email = new_user['email']
         signin.click_next(expect='verify')
-        signin.register_password = user.password
-        signin.verify_password = user.password
-        assert signin.verify_password == user.password, 'verify password getter failed'
+        signin.register_password = new_user['password']
+        signin.verify_password = new_user['password']
+        assert signin.verify_password == new_user['password'], 'verify password getter failed'
```
Comment (bobsilverberg) [inline tests/check_sign_in.py]: Or this.

**Comment by bobsilverberg** [comment]
Looks good, @davehunt. I like the change to use personatestuser. I'm not sure about those asserts that are just asserting what we set in the previous line. Other than that I'm r+. All the tests pass for me locally.

File: tests/check_add_email.py (hunk 1 above)
Comment (davehunt) [inline tests/check_add_email.py]: I believe so. These tests are for the page object, so it's important that setting the value was successful.

**Comment by bobsilverberg** [comment]
@davehunt What is the status of this PR? Are you still waiting on a post-beta version of pytest-selenium for this?

**Comment by davehunt** [comment]
Yep, I haven't forgotten about this. I've rebased it though, to keep it functional. Once pytest-selenium is released, we can update the requirements and merge. I'm currently still testing pytest-selenium with bedrock and mcom-tests. Once I'm happy with those I'll release v1.0.

**Comment by davehunt** [comment]
I've added a version bump, once Travis reports back I'll merge and go ahead with the release.

**Pull request merged**